server/handlers/rawCaptureHandler.js

```diff
@@ -38,6 +54,31 @@ const rawCaptureSchema = Joi.object({
   bulk_pack_file_name: Joi.string(),
 }).unknown(false);
 
+const rawCaptureUpdateSchema = Joi.object({
```

Comment:
The update schema should reflect the exact fields that we want updateable, rather thank doing that in the repository